### ISSUE TITLE
Why no softmax for prediciton (eg: Notebook: 3)

### ISSUE THREAD

**Issue by macabdul9** [bentrevett/pytorch-seq2seq#127]
prediction = self.fc_out(torch.cat((output, weighted, embedded), dim = 1))

Shouldn't here we also use softmax ?

**Comment by bentrevett** [comment]
The loss function used, `CrossEntropyLoss`, does both the softmax and the loss calculation.

See: https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

**Comment by macabdul9** [comment]
Thanks. Closing the issue
